Dockerfile

```
@@ -0,0 +1,45 @@
+# use the official Bun image
+# see all versions at https://hub.docker.com/r/oven/bun/tags
+FROM oven/bun:1 AS base
+WORKDIR /usr/src/app
+
+RUN apt update
```

Hadolint (2.12.0)

[warning] Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead

(DL3027)